util/reggen/multi_register.py

```diff
@@ -165,7 +165,7 @@ def __init__(self,
             self.dv_compact = False
 
     def next_offset(self, addrsep: int) -> int:
-        return self.offset + len(self.regs) * addrsep
+        return self.offset + self.count * addrsep
```

Comment:
This isn't a correct change. Unlike `len(self.regs)`, `self.count` does not take into account that the multi-register is compact.

Reply:
Ok, that makes sense. Sorry for the noise. It's not very clear to outsiders how to consume this format...

Reply:
No worries, and I totally understand!

Getting our work to be readily consumed by outsiders is one of the major TODOs, you might say. There was a long run to develop the repo to have production-quality IP for earlgrey, and efforts were focused on insiders. Since that goal was reached, there is more ongoing work towards expanding the community and providing utility out-of-tree. For that, we've got lots to do yet, though!